R/splitOn.R

```diff
@@ -102,8 +108,9 @@ setGeneric("splitOn",
                 standardGeneric("splitOn"))
 
 # This function collects f (grouping variable), MARGIN, and 
-# use_names and returns them as a list.
-.norm_args_for_split_by <- function(x, f, MARGIN = NULL, use_names = TRUE, ...){
+# use.names and returns them as a list.
+.norm_args_for_split_by <- function(x, f, MARGIN = NULL, use.names = use_names,
```

Comment:
by the way, why do we use `MARGIN` and not `margin`?

Reply:
MARGIN is used in base R. That is why we chose it. However, we will rename MARGIN in another PR

Reply:
We have also parameters `*.FUN`. However, I think that we could stick with lowercase in parameter names

Reply:
Ok